tests/test_pytypes.py

```diff
@@ -5,6 +5,18 @@
 from pybind11_tests import debug_enabled
 
 
+class Hashable(object):
```

Comment:
These two test classes are only used in `test_hash()`. Move them into the scope of the function.

Reply:
Done.